tests/render/test_EmailRenderer.py

```diff
@@ -18,22 +24,66 @@ def test_EmailRenderer_get_report_element():
 
 @pytest.mark.unit
 def test_EmailRenderer_render(v1_checkpoint_result):
+    validation_result = ExpectationSuiteValidationResult(
+        success=True,
+        meta=ExpectationSuiteValidationResultMeta(
+            **{
+                "active_batch_definition": {
+                    "batch_identifiers": {},
+                    "data_asset_name": "taxi_data_1.csv",
+                    "data_connector_name": "default_inferred_data_connector_name",
+                    "datasource_name": "pandas",
+                },
+                "batch_markers": {
+                    "ge_load_time": "20220727T154327.630107Z",
+                    "pandas_data_fingerprint": "c4f929e6d4fab001fedc9e075bf4b612",
+                },
+                "batch_spec": {"path": "../data/taxi_data_1.csv"},
+                "checkpoint_name": "single_validation_checkpoint",
+                "expectation_suite_name": "taxi_suite_1",
+                "great_expectations_version": "0.15.15",
+                "run_id": {
+                    "run_name": "20220727-114327-my-run-name-template",
+                    "run_time": "2022-07-27T11:43:27.625252+00:00",
+                },
+                "validation_time": "20220727T154327.701100Z",
+            }
+        ),
+        statistics={"successful_expectations": 3, "evaluated_expectations": 3},
+        results=[],
+        suite_name="my_suite",
+    )
+    data_docs_pages = {"local_site": "file://localsite/index.html"}
+
     email_renderer = EmailRenderer()
-    _, raw_html = email_renderer.render(checkpoint_result=v1_checkpoint_result)
-    html_blocks = raw_html.split("\n")
-
-    assert html_blocks == [
-        "<p><strong><h3><u>my_bad_suite</u></h3></strong></p>",
-        "<p><strong>Batch Validation Status</strong>: Failed ❌</p>",
-        "<p><strong>Expectation Suite Name</strong>: my_bad_suite</p>",
-        "<p><strong>Data Asset Name</strong>: my_first_asset</p>",
-        "<p><strong>Run ID</strong>: __no_run_id__</p>",
-        "<p><strong>Batch ID</strong>: my_batch</p>",
-        "<p><strong>Summary</strong>: <strong>3</strong> of <strong>5</strong> expectations were met</p><br><p><strong><h3><u>my_good_suite</u></h3></strong></p>",  # noqa: E501
-        "<p><strong>Batch Validation Status</strong>: Success 🎉</p>",
-        "<p><strong>Expectation Suite Name</strong>: my_good_suite</p>",
-        "<p><strong>Data Asset Name</strong>: __no_asset_name__</p>",
-        "<p><strong>Run ID</strong>: my_run_id</p>",
-        "<p><strong>Batch ID</strong>: my_other_batch</p>",
-        "<p><strong>Summary</strong>: <strong>1</strong> of <strong>1</strong> expectations were met</p>",  # noqa: E501
-    ]
+
+    html_blocks = email_renderer.render(
+        validation_result=validation_result,
+        data_docs_pages=data_docs_pages,
+        validation_result_urls=["file://localsite/index.html"],
```

Comment:
Non-blocking, but it may be worth updating this to a url-encoded string so that we can demonstrate the `unquote` call